quickstarts/microsoft.search/azure-search-create-private-endpoint/README.md

```diff
@@ -0,0 +1,10 @@
+# Azure Cognitive Search service
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.search%2Fazure-search-create-private-endpoint%2Fazuredeploy.json)
+[![Deploy To Azure US Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.search%2Fazure-search-create-private-endpoint%2Fazuredeploy.json)
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.search%2Fazure-search-create-private-endpoint%2Fazuredeploy.json)
+
+This template creates a new Azure Cognitive Search Service with a private endpoint. The [quickstart](https://docs.microsoft.com/azure/search/search-get-started-arm) article describes how to deploy the template.
```

Comment:
service is lower case.   regarding phrasing, "agency" phrasing is a little odd.  The template doesn't create anything, and the customer isn't deploying the template.  The link should be relative, not absolute.

I would say:  Use this template to create an Azure Cognitive Search service that's configured for connections over a private endpoint. See [Quickstart: Deploy Cognitive Search using an Azure Resource Manager template](search-get-started-arm.md) for help on template deployment.